seq2science/rules/get_genome.smk

```diff
@@ -118,3 +127,31 @@ rule decoy_transcripts:
     shell:
          ("cpulimit --include-children -l {threads}00 -- " if config.get("cpulimit", True) else " ") +
          "sh {params.script} -j {threads} -g {input.genome} -a {input.gtf} -t {input.transcripts} -o $(dirname {output}) > {log} 2>&1"
+
+
+@lru_cache(maxsize=None)
+def has_annotation(assembly):
+    """
+    returns True/False on whether or not the assembly has an annotation.
+    """
+    # check if genome providd by user or already downloaded, if so check if the annotation came along
+    if all(os.path.exists(f"{config['genome_dir']}/{assembly}.{extension}") for extension in config["genome_types"]):
+        return os.path.exists(f"{config['genome_dir']}/{assembly}.annotation.gtf")
+
+    if "provider" in config:
+        providers = config["provider"]
```

Comment:
should this not be a list?

`providers = [config["provider"]]`

Reply:
correct